### PR TITLE
Ctools plugins training

### PR DESCRIPTION
**Pull request by liatsade**
https://www.gitbook.com/book/amitaibu/the-gizra-way/edit#/edit/ctools-plugins/ctools_plugins.md

### CODE REVIEW

File: ctools_plugins.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  </div>
+</footer>
+```
+
+That’s all we need. Delete the rest of the original code in ```page.tpl.php```.
+
+
+### 4 - Create Panel page
+In drupal site, go to ```/admin/structure/pages``` and create a page. Give it title and path, choose 1 column layout and leave content empty for now. Later we will add the plugin to the content area.
+Save the panel page and export it by Features. Put the module in ```/modules/custom/MYMODULE```. Don’t forget to enable the module.
```
Comment (liatsade) [inline ctools_plugins.md]: @amitaibu Is it important to write here few words about add Ctools & Panels to module dependencies?
If yes - in which file we define the dependencies?

**Comment by liatsade** [comment]
@RachelBaram please review.

**Comment by amitaibu** [comment]
If ready for review, please remove `WIP` from title.

**Comment by RachelBaram** [comment]
@liatsade Proofread. Also, please remember to include the Gitbook link in the top PR comment. I added it for you this time. ;)

File: ctools_plugins.md [hunk 2]
```diff
@@ -234,30 +235,28 @@ The `$block` variable will get the renderable content of the plugin.
...
 
 Check out the final code [here](https://github.com/Gizra/dynamic_example/tree/lesson1/dynamic_example/modules/custom/dynamic_example_lesson/plugins/content_types/companies_list).
 
-The plugin displays list view of companies, based of this markup: 
+The plugin displays list view of companies, based on this markup: 
 
 ![](list view plugin.png)
 
-As you can see, in this markup we have container div and items inside it. Each item has **image**, **title** and **link** (the title is a link to full node). So when we build the plugin, it should have two template files: `compatnies_list.tpl.php` holds the HTML for the container div, and `companies_item.tpl.php` holds the HTML for the item itself.
+As you can see, in this markup we have container div and items inside it. Each item has **image**, **title** and **link** (the title is a link to full node). So, when we build the plugin, it should have two template files: `companies_list.tpl.php` holds the HTML for the container div, and `companies_item.tpl.php` holds the HTML for the item itself.
```
Comment (RachelBaram) [inline ctools_plugins.md:259]: @liatsade Please double check that this fix in the file name is OK.

Comment (liatsade) [reply ctools_plugins.md:259]: Sure. the "t" was a mistake there. Tnx.

**Comment by liatsade** [comment]
Thanks @RachelBaram .
@amitaibu can you merge it to master?

**Comment by amitaibu** [comment]
@liatsade if ready for review, please add the `ready for review` label

**Comment by amitaibu** [comment]
@liatsade needs a re-roll

**Comment by liatsade** [comment]
@amitaibu I fixed the conflict. Hope I fix it right :-)

**Comment by amitaibu** [comment]
I didn't review it properly yet - just went over it quickly. But I think it's already ready to get in, and we can do a "post-mortem" review.

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request merged**